homeassistant/components/cover/isy994.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for ISY994 covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.isy994/
+"""
+import logging
+from typing import Callable  # noqa
+
+from homeassistant.components.cover import CoverDevice, DOMAIN
+from homeassistant.components.isy994 import (ISYDevice, NODES, PROGRAMS, ISY,
```

Comment:
I might be missing the point too :smile: was just told at a point not to do this

Cant find a good reference at the moment, but you might try [this](https://docs.python.org/3/faq/programming.html#what-are-the-best-practices-for-using-import-in-a-module)

Our own BD(FL) might know.